src/App.jsx

```diff
@@ -18,6 +19,7 @@ const [articles, setArticles] = useState([])
   <div className='content'>
   <Routes>
     <Route path="/" element={<Articles articles={articles} setArticles={setArticles} />} />
+    <Route path="/api/articles/:article_id" element={<><IndividualArticle /></>}/>
```

Comment:
You seem to be choosing to render your component inside a React fragment, is the fragment necessary, `element={<IndividualArticle />}`